deploy/operator.yaml

```diff
@@ -32,6 +32,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: IMAGE_NAME
```

Comment:
I would probably use IMAGE_REFERENCE or something (to avoid `name`), but this works as well

Reply:
since I have to reupload, I'll change this.